cmd/docker/builder.go

```diff
@@ -18,34 +18,30 @@ const (
 	builderDefaultPlugin = "buildx"
 	buildxMissingWarning = `DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
             Install the buildx component to build images with BuildKit:
-            https://docs.docker.com/go/buildx/
-`
```

Comment:
Had to remove the extra newline here, otherwise it was complaining if we used `Println` with a message that contains a newline at the end.